hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/webapp/dao/ContainersInfo.java

```diff
@@ -46,4 +46,7 @@ public ArrayList<ContainerInfo> getContainers() {
     return container;
   }
 
+  public void addAll(ArrayList<ContainerInfo> containersInfo) {
```

Comment:
Maybe just make this `Collection<ContainerInfo>`

Reply:
Thanks for helping to review the code, I will modify the code.